cmd/spire-agent/cli/api/api_test.go

```diff
@@ -100,10 +102,12 @@ func TestFetchJWTCommand(t *testing.T) {
   {
     "svids": [
       {
+		"hint": "external",
         "spiffe_id": "spiffe://domain1.test",
         "svid": "%s"
       },
       {
+		"hint": "",
```

Comment:
This line is using tabs when the rest of the file uses spaces. Can you changes this to spaces match the rest of the file?

Reply:
435498e74f5a8c107b12168c25e651557f46e405